src/commands/MineSweeperCommand.ts

```diff
@@ -0,0 +1,48 @@
+import { Message, MessageEmbed } from "discord.js";
+import Command from "../abstracts/Command";
+import { EMBED_COLOURS } from "../config.json";
+import MineSweeperService from "../services/MineSweeperService";
+import StringUtils from "../utils/StringUtils";
+
+class MineSweeperCommand extends Command {
+	constructor() {
+		super(
+			"minesweeper",
+			"Generates a minesweeper game."
+		);
+	}
+
+	async run(message: Message, args: string[]) {
+		const embed = new MessageEmbed();
+		const mineSweeperService = MineSweeperService.getInstance();
+
+		if (!args[0]) args[0] = "easy";
+
+		interface DifficultyLevel {
+			[key: string]: {ratio: number}
+		}
```

Comment:
Don't define an interface within a method or class. Also, refer to the comment below about changing the signature.